src/main/kotlin/org/jitsi/nlj/transform/node/incoming/IncomingStatisticsTracker.kt

```diff
@@ -159,12 +169,14 @@ class IncomingSsrcStats(
         }
     }
 
-    fun getSnapshot(): Snapshot? {
+    fun getSnapshot(onlyActive: Boolean = false): Snapshot? {
         synchronized(statsLock) {
-            if (!activitySinceLastSnapshot) {
-                return null
+            if (onlyActive) {
+                if (!activitySinceLastSnapshot) {
+                    return null
+                }
+                activitySinceLastSnapshot = false
```

Comment:
Shouldn't this be outside the `if (onlyActive) {` block?

Reply:
No, the idea is to prevent the code in the bridge that generates stats from resetting the `activitySinceLastSnapshot` flag (which is only used for RR generation). It's pretty awkward. I'll see if I can make it more clear.

Reply:
No, the point of this is to get the snapshot for stats or whatever, which shouldn't reset the activity flag.